Application/Frontend/components/FriendsColumn/Fetch_chat.tsx

```diff
@@ -0,0 +1,62 @@
+import React, { useState } from 'react';
+
+function ChatOwnerFinder() {
+  const [channelKey, setChannelKey] = useState('');
+  const [owner, setOwner] = useState('');
+  const [error, setError] = useState('');
+
+  const fetchChatOwner = async () => {
+    setError(''); // Reset error message
+    try {
+      const response = await fetch('/api/get_chat', { // Adjust with your API endpoint
```

Comment:
Although the name of this API endpoint is valid, one of the principles of REST APIs is "uniform interface", and you should rename this endpoint to use a hyphen (-) instead of an underscore (_) in line with addressing this requirement.